app/controllers/api/promos_controller.rb

```diff
@@ -4,21 +4,12 @@ class Api::PromosController < Api::BaseController
 
   api_versions :v1
 
-  def resource
-    super.tap do |af|
-      if story && af.try(:new_record?)
-        af.audio_version_id ||= story.promos.id
-        af.account_id ||= story.account_id
-      end
-    end
-  end
-
-  def resources
-    @promos ||= story.try(:promos_audio)
+  def resources_base
+    story.promos_audio
   end
 
   def story
-    @story ||= Story.find(params[:story_id]) if params[:story_id]
+    @story ||= Story.find(params[:story_id])
```

Comment:
noting that this controller previously unnecessarily accommodated promos without a story specified.
this is not allowed by the routes, so this is an appropriate assumption.